unlock_indicator.c

```diff
@@ -272,13 +272,36 @@ xcb_pixmap_t draw_image(uint32_t *resolution) {
     }
 
     if (xr_screens > 0) {
+        /* This indicates if we have to draw the indicator to our current screen*/
+        bool draw_indicator = true;
+
+        /* This keeps track of the screens drawn to */
+        bool drawn_to[xr_screens];
```

Comment:
I don’t think `sizeof(bool) != 1` on any Unix platform on which i3 runs, but strictly speaking you’re right. I don’t mind either way.